components/salesforce/sources/new-object-instant/new-object-instant.js

```diff
@@ -0,0 +1,36 @@
+const startCase = require("lodash/startCase");
+
+const common = require("../../common-instant");
+
+module.exports = {
+  ...common,
+  name: "New Object (Instant)",
+  key: "salesforce-new-object-instant",
+  description: "Emit an event when an object is created",
```

Comment:
Same feedback as on the other PR re: clarifying that the source allows you to select your target object.

For context, we're planning to display the description more prominently, so you see it alongside the event source's title.

Reply:
It would be good to clarify why users should choose this vs. the non-instant trigger in the description.

Reply:
Will do. I was also thinking of adding a `README` with more details, would that make sense?